article.php

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/css/header.css">
+    <link rel="stylesheet" href="assets/css/footer.css">
+    <link rel="stylesheet" href="assets/css/global.css">
+    <link rel="stylesheet" href="assets/css/article.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@200&display=swap" rel="stylesheet">
+    <link rel="icon" type="image/x-icon" href="assets/img/logo_fav.svg">
+    <link rel="mask-icon" href="assets/img/logo_fav.svg">
+    <title>Accueil</title>
+</head>
+
+<body>
+    <?php require_once('header.php'); ?>
+    <main>
+        <p><a id="go_back" href="index.php">Revenir à l'accueil</a></p>
+        <?php require_once('assets/db/article.php'); ?>
+
+        <section class="body_text">
+            <h1 id="article_title">
+                <?php echo $title ?>
+            </h1>
+
+            <p class="article_P">
+                <?php echo $description ?>
+            </p>
+        </section>
+        <section id="img_section">
+            <img id="article_img" src="<?php echo $coverImg ?>" alt="bruh" />
+        </section>
+
+        <section class="body_text">
+            <h2 class="body_title">Introduction</h2>
+            <p class="article_P">
+                <?php echo $introduction ?>
+            </p>
+
+            <h2 class="body_title">Le détail</h2>
+            <p class="article_P">
+                <?php echo $detail ?>
+            </p>
+        </section>
+
+        <section id="author_section">
```

Comment:
Why don't you replace this section by "a" tag with authors href instead of using javascript ?

Reply:
![image](https://github.com/WildCodeSchool/blog-e-boulga/assets/25879136/9b564bb4-8a28-4af9-9dd9-9b78c2e396a3)

![image](https://github.com/WildCodeSchool/blog-e-boulga/assets/25879136/d90f767f-a607-4d53-9a49-b682effd6e34)

Changed ✅ 
Appliqué au prochain push 👍